std/datetime/systime.d

```diff
@@ -31,6 +31,38 @@ $(TR $(TD Conversion) $(TD
 +/
 module std.datetime.systime;
 
+/// Get the current time as a $(LREF SysTime)
+@safe unittest
+{
+    import std.datetime.timezone : LocalTime;
+    SysTime today = Clock.currTime();
+    assert(today.timezone is LocalTime());
+}
+
+/// Construct a $(LREF SysTime) from a ISO time string
+@safe unittest
+{
+    import std.datetime.timezone : UTC;
+
+    auto st = SysTime.fromISOExtString("2018-01-01T10:30:00Z");
+    assert(st == SysTime(DateTime(2018, 1, 1, 10, 30, 0), UTC()));
+}
+
+/// Make a specific point in time in the New York timezone
+@safe unittest
+{
+    import std.datetime.timezone : SimpleTimeZone;
+
+    auto ny = SysTime(
+        DateTime(2018, 1, 1, 10, 30, 0),
```

Comment:
Maybe add a comment saying what time this corresponds to. It's kind of belaboring the obvious, but I think it wouldn't hurt.